.circleci/config.yml

```diff
@@ -186,7 +195,6 @@ jobs:
             forge fmt --check
 
 workflows:
-  version: 2
```

Comment:
This key is deprecated: https://github.com/SchemaStore/schemastore/blob/91d3af2df2a62f93faaf1e97c4b642557bf81440/src/schemas/json/circleciconfig.json#L1443